app/models/forms/bulk_upload_lettings/prepare_your_file.rb

```diff
@@ -26,7 +26,19 @@ def next_path
       end
 
       def template_path
-        "/files/bulk-upload-lettings-template-v1.xlsx"
+        "/files/bulk-upload-lettings-template-2023-24.xlsx"
+      end
+
+      def old_template_path
+        "/files/bulk-upload-lettings-template-2022-23.xlsx"
+      end
+
+      def specification_path
+        if year <= 2022
```

Comment:
seems more stable than an == but == would probably suffice - year is not guarded atm it's just a param in the year but if that changes this could change too